config-model/src/main/java/com/yahoo/vespa/model/container/ContainerCluster.java

```diff
@@ -114,7 +114,6 @@
     public static final String STATE_HANDLER_CLASS = "com.yahoo.container.jdisc.state.StateHandler";
     public static final String LOG_HANDLER_CLASS = com.yahoo.container.handler.LogHandler.class.getName();
     public static final String DEFAULT_LINGUISTICS_PROVIDER = "com.yahoo.language.provider.DefaultLinguisticsProvider";
-    public static final String CMS = "-XX:+UseConcMarkSweepGC -XX:MaxTenuringThreshold=15 -XX:NewRatio=1";
```

Comment:
`CMS` is used in YahooContainerModelAmender

Reply:
hrmf, fixed